src/main/scala/scalismo/faces/io/ply/PlyMeshWriter.scala

```diff
@@ -88,6 +88,11 @@ private[io] case class PlyMeshWriter(url: String,
 
     val _vNormals = normals.flatMap {
       case n: SurfacePointProperty[EuclideanVector[_3D]] => Some(new VertexNormal(n))
+      case n: MappedSurfaceProperty[EuclideanVector[_3D],_] => {
```

Comment:
I think the type should be restricted to EuclideanVector[_3D] for the second parameter. But this is already implied by the parameter.... so `case n: MappedSurfaceProperty[_,_] =>  ....` should work and  is more general.